### PR TITLE
chore: upgraded integration tests to use DAL API

### PR DESCRIPTION
**Pull request by kevin-dp**

### CODE REVIEW

File: integration_tests/satellite_client/package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "@types/uuid": "^9.0.0",
-    "electric-sql": "git+https://github.com/electric-sql/typescript-client.git#cbc83d2d8b229338bd46ff93d591a1387f7dabea",
+    "electric-sql": "git+https://github.com/electric-sql/typescript-client.git#bc0c8e540ebb1fc08c21faf466f3359fa2dd9aaa",
```
Comment (define-null) [inline integration_tests/satellite_client/package.json:18]: This commit does not seem to be available. Have you merged it to the main branch already? If that's the case - the whole branch going to be automatically deleted, thus causing this error.

File: integration_tests/satellite_client/src/client.ts [hunk 2]
```diff
@@ -42,16 +44,33 @@ export const read_migrations = (migration_file: string) => {
   return json_data.migrations
 }
 
+const dbSchema = {
```
Comment (define-null) [inline integration_tests/satellite_client/src/client.ts:47]: Do we have a simple way to generate this schema somehow?

File: integration_tests/satellite_client/package.json (hunk 1 above)
Comment (kevin-dp) [reply integration_tests/satellite_client/package.json:18]: No not yet, waiting for reviews before merging it.

File: integration_tests/satellite_client/src/client.ts (hunk 2 above)
Comment (kevin-dp) [reply integration_tests/satellite_client/src/client.ts:47]: Yes, this schema can be generated directly from the Prisma schema or defined manually like here.

**Comment by icehaunter** [comment]
Superseded by #123

**Pull request closed without merging**